hack/create-bundle

```diff
@@ -17,42 +17,15 @@ OPERATOR_IMAGE=$2
 BUNDLE_IMAGE=$3
 INDEX_IMAGE=$4
 
-# Prepare output dir
-mkdir -p opm-bundle
-pushd opm-bundle
+pushd config
 
-mkdir -p manifests metadata
-
-cat - <<EOF > metadata/annotations.yaml
-annotations:
-  operators.operatorframework.io.bundle.channel.default.v1: 4.9
-  operators.operatorframework.io.bundle.channels.v1: 4.9
-  operators.operatorframework.io.bundle.manifests.v1: manifests/
-  operators.operatorframework.io.bundle.mediatype.v1: registry+v1
-  operators.operatorframework.io.bundle.metadata.v1: metadata/
-  operators.operatorframework.io.bundle.package.v1: aws-efs-csi-driver-operator
-EOF
-
-cat - <<EOF > Dockerfile
-FROM scratch
-LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
-LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
-LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
-LABEL operators.operatorframework.io.bundle.package.v1=aws-efs-csi-driver-operator
-LABEL operators.operatorframework.io.bundle.channels.v1=4.9
-LABEL operators.operatorframework.io.bundle.channel.default.v1=4.9
-COPY manifests /manifests/
-COPY metadata/annotations.yaml /metadata/annotations.yaml
-EOF
-
-MANIFEST=manifests/aws-efs-csi-driver-operator.clusterserviceversion.yaml
-cp -v ../../manifests/4.*/aws-efs*.yaml $MANIFEST
+MANIFEST=manifests/4.9/aws-efs-csi-driver-operator.clusterserviceversion.yaml
 
 # Replace images in the manifest - error prone, needs to be in sync with image-references.
 sed -i $MANIFEST -e "s~quay.io/openshift/origin-aws-efs-csi-driver-operator:latest~$OPERATOR_IMAGE~" -e "s~quay.io/openshift/origin-aws-efs-csi-driver:latest~$DRIVER_IMAGE~"
```

Comment:
This will replace image names in the git work tree, while before this PR this script modified a copy.

Reply:
yes, it will. Couple of options I am thinking:

1. create a separate directory like `opm-bundle` that we were creating before and use that as place where all changes are made.
2. Actually revert the `$MANIFEST` after building the index image via `git checkout`. I wonder if one will want to manually inspect the CSV file.

Reply:
I'd prefer if I could make local changes to the manifest and try it out with the script, without risking I loose my changes with `git checkout` and/or accidentally committing any changes made by the script. Copy is IMO the safest form.

Reply:
ack. `create-bundle` will create local copy in `hack` folder for now.